azure_token.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+api=$1
+method=${2:-GET}
+
+token=$(curl -X POST -H 'Content-Type: application/x-www-form-urlencoded' \
+-d 'grant_type=client_credentials&client_id=366fab79-6c61-4a24-8c2c-b2be2f48fe35&resource=2ff814a6-3304-4ab8-85cb-cd0e6f879c1d&client_secret=7OA7Q~RL5pELYji-kU59UfStGoRVGco8f20~K' \
```

Comment:
the secret should not be here 

even though it's just our test tenant and test client